controller/volume_controller.go

```diff
@@ -3621,15 +3621,16 @@ func (c *VolumeController) createEngine(v *longhorn.Volume, currentEngineName st
 	}
 
 	if v.Spec.FromBackup != "" && v.Status.RestoreRequired {
-		backupVolumeName, backupName, err := c.getInfoFromBackupURL(v)
+		remoteBackupVolumeName, backupVolumeName, backupName, err := c.getBackupVolumeInfo(v)
 		if err != nil {
-			return nil, errors.Wrapf(err, "failed to get backup volume when creating engine object of restored volume %v", v.Name)
+			return nil, errors.Wrapf(err, "failed to get backup volume information for restoring volume %v", v.Name)
 		}
+
 		engine.Spec.BackupVolume = backupVolumeName
 		engine.Spec.RequestedBackupRestore = backupName
 
-		log.Infof("Creating engine %v for restored volume, BackupVolume is %v, RequestedBackupRestore is %v",
-			engine.Name, engine.Spec.BackupVolume, engine.Spec.RequestedBackupRestore)
+		log.Infof("Creating engine %v for restored volume %v (backup volume %v, remote backup volume %v, restoring backup %v)",
+			engine.Name, v.Name, engine.Spec.BackupVolume, remoteBackupVolumeName, engine.Spec.RequestedBackupRestore)
 	}
```

Comment:
```suggestion
		log.WithFields(logrus.Fields{
			"backupVolume": engine.Spec.BackupVolume,
			"backupVolumeRemote": remoteBackupVolumeName,
			"engine":       engine.Name,
			"restoreBackup": engine.Spec.RequestedBackupRestore,
		}).Info("Creating engine for restored volume)")
	}
```
No need `v.Name` since it's already included in `log`.

Reply:
Modified. Thanks.